twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -54,6 +54,11 @@ def check(self, instance):
         if not self.config.username or not self.config.password:
             raise Exception('The Twistlock check requires both a username and a password')
 
+        self.instance_tags = instance.get('tags')
+        self.project = instance.get('project')
+        if self.project:
+            self.instance_tags.append("project:{}".format(self.project))
```

Comment:
You shouldn't do that. Change the `Config` class to append to its tags attribute instead.